apps/api/src/app/inbox/utils/notification-mapper.ts

```diff
@@ -44,16 +44,25 @@ const mapSingleItem = ({
     primaryAction: primaryCta && {
       label: primaryCta.content,
       isCompleted: actionType === ButtonTypeEnum.PRIMARY && actionStatus === MessageActionStatusEnum.DONE,
+      redirect: {
+        url: primaryCta.url,
+        target: primaryCta.target,
+      },
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** please copy the same changes to this file: `packages/js/src/ws/socket.ts:27`, this is unfortunate that we have it duplicated but yeah :(